### ISSUE TITLE
Large filesize

### ISSUE THREAD

**Issue by laurencedawson** [orhanobut/hawk#171]
Hi,
I'm just starting to use Hawk but I noticed it's adding around 3mb extra to my APK due to using Facebook conceal. Is it possible to remove this?

Thanks

**Comment by tanweijiu** [comment]
agree，     can  be a choice for Facebook conceal?

**Comment by rubengees** [comment]
The latest version of conceal is much smaller.
[This](https://github.com/orhanobut/hawk/commit/111c98866d5f192797616b067959c6fd6c6342be) commit updates it, but there is no release of Hawk yet, which includes this change. If you still want to update to the latest version, you can do it like this:

```groovy
compile 'com.facebook.conceal:conceal:1.1.3@aar'
compile 'com.orhanobut:hawk:2.0.0-Alpha'
```

**Comment by orhanobut** [comment]
I just released a new version (2.0.1) for this issue which contains @rubengees PR. and thanks for the answers @rubengees
